util/cron/common-native-gpu.bash

```diff
@@ -6,6 +6,9 @@ source $CWD/common.bash
 # number as a parameter to the script.
 # source /cray/css/users/chapelu/setup_system_llvm.bash
 
+# Prevent which LLVM from being set by this variable, so it can be overridden
+# later to use ROCM LLVM.
+unset CHPL_LLVM_CONFIG
 export CHPL_LLVM=system
```

Comment:
Won't this also effect CHPL_GPU=cuda configs?

Reply:
Yeah it would -- I was thinking it'd be safer to do that than try and modify each ROCM script and risk missing one, and also to avoid duplication. Honestly though I'm not too likely to miss one, and duplication for something going away soon is okay, so I'll switch it over.